src/ngraph/runtime/cpu/cpu_emitter.cpp

```diff
@@ -1652,6 +1652,14 @@ namespace ngraph
             void CPU_Emitter::EMITTER_DECL(ngraph::op::Reshape)
             {
                 auto reshape = static_cast<const ngraph::op::Reshape*>(node);
+                auto annotation = reshape->get_op_annotations();
+                if (annotation && annotation->get_in_place_oi_pairs().size() > 0)
+                {
```

Comment:
Safer to check if out[0].get_name() == args[0].get_name() 
Memory allocation doesn't guarantee in-place buffers. Especially if args[0] is an input parameter.

Reply:
second that